apps/projects/app/components/App/App.js

```diff
@@ -102,16 +102,9 @@ const getURLParam = param => {
  */
 const getToken = async code => {
   console.log('getToken entered')
-
-  // TODO: Manage when server does not respond
-  try {
-    let response = await fetch(`${AUTH_URI}/${code}`)
-    let json = await response.json()
-    if (json.token) return json.token
-    else throw Error(`${json.error}`)
-  } catch (e) {
-    console.error('Error from Authentication server:', e)
-  }
+  const response = await fetch(`${AUTH_URI}/${code}`)
+  const json = await response.json()
+  return json.token
 }
```

Comment:
Not a big deal since we probably will refactor this part to make use of hooks and reusability, just wondering if we should still keep try-catching eventual fetch errors, since it is a common scenario on poor connectivity conditions